Workbooks/Azure Monitor - Getting Started/Resource Picker/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Picking a set of resources to analyze in workbooks
+
+The `Resource Picker` template gets you started with subscription, resource group and resource parameters to set up the input context of your workbook. The default parameters are set to pick virtual machines, but you can configure it to pick any type of resource. The template also has a ARG query control that shows you how to use the parameters in your analyses.
+
+![Image](Full.png)
```

Comment:
should this be a full path to the image in github?

Reply:
Relative path should work.